dhall-openapi/dhall-openapi.cabal

```diff
@@ -1,6 +1,6 @@
 Cabal-Version:  >=1.11
 Name:           dhall-openapi
-Version:        1.0.2
+Version:        1.0.3
```

Comment:
You don't need to bump the version number.  We compute the new version for each package when we publish releases